sensor_msgs/JointState joints
bool neutral
---
bool success